VHT/roundrobin.erl
# Rebar3 and Erlang 22.1 (erts-10.5) upgrade

[//]: * (If not ready for merging, indicate the reason after the # on the next line.)
#
#### Card(s)
[//]: * (Link to the PivotalTracker or LeanKit cards.)
- [ConfigurationManagement upgrade to rebar3](https://www.pivotaltracker.com/story/show/171130194)

#### Related Pull Requests
[//]: * (List pull requests that must be merged before this one.)
- [qmcl](https://github.com/VHT/qmcl/pull/22)
- [ConfigurationManagement](https://github.com/VHT/ConfigurationManagement/pull/619)

#### Changes Made
1. Rebar3
2. Use Hex Packages

#### Test Procedure
[//]: * (Use Cucumber format, if possible.)
```
Eunits and ct will run with builds and Pipelines will verify systems are working as expected
```
